packages/cadl-ranch-specs/http/payload/media-type/main.tsp

```diff
@@ -0,0 +1,55 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+/**
+ * Test the payload with different media types and different types of the payload itself.
+ */
+@supportedBy("dpg")
+@scenarioService("/payload/media-type")
+namespace Payload.MediaType;
+
+@scenario
+@scenarioDoc("""
+  When the request body and response body are both strings, the different media types would lead to different payloads.
+  """)
+@operationGroup
+@route("/stringBody")
+namespace StringBody {
+  /**
+   * Expected request body is a string '{cat}'.
+   */
```

Comment:
Should you do `@scenario` and `@scenarioDoc` on each API?

I guess the comment here won't show up in dashboard. And it be hard for dev to figure out what is the expected request/response.